core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.deploy.history
```

Comment:
If it's occurred from here I'll remove. If not, it might make other PRs be broken, so I might take it carefully.

Reply:
Since `FsHistoryProvider.scala` is one of the main target of this development (I mean not the PR) it would be good to clean it up somewhere. What do you think where should we do it?

Reply:
I guess it's OK to do with minor PR, but commiter could finally judge the worth. (treat my comment as 2 cents)

Reply:
I agree with this resolution. Such way we separate the concerns. I've created https://github.com/apache/spark/pull/26436, in case of disagreement it can be just dropped.